gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/FileExporter.java

```diff
@@ -216,8 +216,6 @@ public String exportCgmesZipFile(SweData sweData, Map<String, ByteArrayOutputStr
                 }
                 is.close();
             }
-            zipOs.close();
```

Comment:
I believe those `close` methods were explicitely called here because we can't use `toByteArray` while the ByteArrayOutputStream is still open.

Reply:
:/ not sure here but I'll put them back just in case